usaspending_api/api_contracts/contracts/v2/download/transactions.md

```diff
@@ -0,0 +1,99 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Award Download [/api/v2/download/transactions/]
+
+This endpoint is used by the Advanced Search download page.
+
+## POST
+
+This route sends a request to the backend to begin generating a zipfile of transaction data in CSV form for download.
+        
++ Request (application/json)
+    + Attributes (object)
+        + `columns` (optional, array[string])
+        + `filters` (required, Filters, fixed-type)
+        + `file_format` (optional, string)
+            + Default: `csv`
+        + `limit` (optional, number)
+    + Body
+
+            {
+                "filters": {
+                    "keywords": ["Defense"]
+                },
+                "columns": [
+                    "assistance_transaction_unique_key",
+                    "award_id_fain",
+                    "modification_number",
+                    "award_id_uri",
+                    "sai_number",
+                    "total_funding_amount"
+                ]
+            }
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `file_name` (required, string) 
+            Is the name of the zipfile containing CSVs that will be generated (file_name is timestamp followed by `_transactions` or `_awards`).
+        + `message` (required, string, nullable) 
+            A human readable error message if the `status` is `failed`, otherwise it is `null`.
+        + `seconds_elapsed` (required, string, nullable) 
+            Is the time taken to genereate the file (if `status` is `finished` or `failed`), or time taken so far (if `running`).
+        + `status` (required, enum[string]) 
+            A string representing the current state of the CSV generation request.
+            + Members
+                + `failed`
+                + `finished`
+                + `ready`
+                + `running`
+        + `total_columns` (required, number, nullable) 
+            Is the number of columns in the CSV, or `null` if not finished.
+        + `total_rows` (required, number, nullable) 
+            Is the number of rows in the CSV, or `null` if not finished.
+        + `total_size` (required, number, nullable) 
+            Is the estimated file size of the CSV in kilobytes, or `null` if not finished.
+        + `url` (required, string) 
+            The URL for the file.
+
+# Data Structures
+
+## Filter Objects
+
+### Filters (object)
++ `award_amounts` (optional, array[string])
```

Comment:
`award_amounts` is an array with an object, e.g. `"award_amounts": [{"lower_bound": 1000000, "upper_bound": 25000000}]`. Similar for `agencies` (e.g. `"agencies": [{type: "awarding", tier: "toptier", name: "Department of Agriculture"}]`